pkg/terra/client/test_helpers.go

```diff
@@ -45,7 +44,7 @@ var minGasPrice = msg.NewDecCoinFromDec("uluna", msg.NewDecWithPrec(1, 3))
 // SetupLocalTerraNode sets up a local terra node via terrad, and returns pre-funded accounts, the test directory, and the url.
 func SetupLocalTerraNode(t *testing.T, chainID string) ([]Account, string, string) {
 	t.Skip("depends on terrad")
-	testdir, err := ioutil.TempDir("", "integration-test")
+	testdir, err := os.MkdirTemp("", "integration-test")
```

Comment:
Could go a step further here with the built in `t.TempDir()`, which is also cleaned up automatically too.
```suggestion
	testdir := t.TempDir()
```